experimental/adaptive-tool/README.md

```diff
@@ -1,57 +1,122 @@
 # Bot Framework Adaptive Tool
-BotFramework adaptive tool is a vscode extension to help developers to improve the fficiency of handering LG/LU files.
+Bot Framework Adaptive Tool is a Visual Studio Code extension that helps developers handle LG (.lg) and LU (.lu) files efficiently.
```

Comment:
>LG (.lg [](start = 91, length = 7)

Does it handle .qna as well?  If not, we should extend it to handle .qna as well.

Reply:
Not support currently. But we have plans to support .qna